docs/samples/hub-state-import-data-sample.json

```diff
@@ -0,0 +1,925 @@
+{
```

Comment:
In general the payload representing the state of an IoT Hub should only include what is necessary to recreate it. This sample of the payload should include the required elements and no extraneous ones.